mechanical/mechanical_integration/README.md

```diff
@@ -1,32 +1,154 @@
-# Mechanical Integration
-This is the final mechanical integration of all of the other subassemblies to create the final rover.
 
-<img src="/images/mechanical_integration/final_title.png" width="80%">
 
-## Features
-  * All subassemblies can be combined together with the addition of very few additional components.
+#  1 Mechanical/Structural Assembly
+At this point we have every major sub assembly created and they just need to be attached together to create the rover! In the steps in this document, we will attach all the corner motors and wheels, the rocker-bogie, the differential pivot, and the head.
 
-## Information about module
+<table>
+<tr>
+	<td><img src="../../images/mechanical_integration/differential_pivot.png">
+	</td>
+	<td><img src="../../images/mechanical_integration/head_final.png"></td>
+	<td><img src="../../images/mechanical_integration/wheels_title.png"></td>
+</tr>
+</table>
+<table>
+<tr>
+	<td><img src="../../images/mechanical_integration/rocker_bogie_title.png"></td>
+	<td><img src="../../images/mechanical_integration/finished_body.png"></td>
+</tr>
+</table>
 
-| Author/Creator            | JPL       |
-| --------------            | -------            |
-| Cost without 3D printer   | 56.58              |
-| Cost with 3D printer      | 56.58              |
-| Date Added                | 4/11               |
-| Group/Organization/School | JPL                |
-| Time to build             | 2 hrs              |
+**Figure 1: All subassemblies completed**
 
-## Skills
+## 1.1 Mounting Wheels
```

Comment:
Should make this table two columns. Refer to https://github.com/nasa-jpl/open-source-rover/blob/master/examples/README_example.md